CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - Ensure performance measurement collection is not taken too frequently ([#3221](https://github.com/getsentry/sentry-java/pull/3221))
 - Fix old profiles deletion on SDK init ([#3216](https://github.com/getsentry/sentry-java/pull/3216))
 - Experimental: Add Metrics API ([#3205](https://github.com/getsentry/sentry-java/pull/3205))
+- Force flush metrics when weight is too high ([#3220](https://github.com/getsentry/sentry-java/pull/3220))
```

Comment:
do we need a changelog entry given this is not targeting `main` and the metrics feature is not shipped yet?

Reply:
No we don't. Let me remove this